### PR TITLE
PLUGIN QRCode

### PR DESCRIPTION
**Pull request by ArthurHoaro**
Add an icon in linklist to display links QRCode

### CODE REVIEW

File: plugins/qrcode/qrcode.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="linkqrcode">
+    <a href="http://qrfree.kaywa.com/?l=1&amps=8&ampd=%s" onclick="showQrCode(this); return false;" class="qrcode" data-permalink="%s"><img src="%s/qrcode/qrcode.png" width="13" height="13" title="QR-Code"></a>
```
Comment (virtualtam) [inline plugins/qrcode/qrcode.html]: missing `;` when escaping the ampersands: `&amp;`

**Comment by nicolasdanelon** [comment]
Just a comment... 
```javascript
function showQrCode(caller,loading)
(...)
function remove_qrcode()
```
camel or snake case? choose one :)

File: plugins/qrcode/qrcode.html (hunk 1 above)
Comment (ArthurHoaro) [inline plugins/qrcode/qrcode.html]: Thanks.

**Comment by ArthurHoaro** [comment]
@nicolasdanelon Yep, I didn't review the JS code. I'll take a look.

**Comment by nodiscc** [comment]
Hi everyone, keep in mint that the Javascript code is imported directly from https://github.com/neocotic/qr.js, so any edits would be lost if qr.js is ever updated from upstream. My vote is for not touching the js code unless necessary.

**Comment by ArthurHoaro** [comment]
It's not the library js file but the other one, which call the lib with lazy loading. 
-- 
Arthur Hoaro
http://hoa.ro
- Envoi mobile -

**Comment by nicolasdanelon** [comment]
that's right. i was talking about ```plugins/qrcode/shaarli-qrcode.js ```

**Pull request merged**